docs/PowerAuth-Server-1.6.0.md

```diff
@@ -21,3 +21,13 @@ and `pa_application` or `pa_operation`. Make sure that `pa_operation_application
 existing `pa_application.id` and `pa_operation_application.operation_id` contains references to
 existing `pa_operation.id`. If necessary, manually remove orphaned records in `pa_operation_application`. Consider
 creating a backup before this operation.
+
+### Add activation_id Column
+
+Add a new column `activation_id` to the `pa_operation` table. This column is a foreign key that references
+the `activation_id` column in the `pa_activation` table. Storing the `activation_id` in the `pa_operation` table
+provides several enhancements for developers:
```

Comment:
I would remove the `for developers` words, because this change also impacts users.